libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -698,3 +704,18 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
         list-style: none;
     }
 }
+
+button.gd-button-primary {
+    &.icon-sources.s-sources.gd-button {
+        > span {
+            margin-left: 20px;
+        }
+    }
+
+    &.icon-model.s-model.gd-button,
+    &.icon-cloud-upload.s-load.gd-button {
+        > span {
+            margin-left: 24px;
```

Comment:
![image](https://user-images.githubusercontent.com/67769750/103207355-a7f45e80-4930-11eb-981f-29f2e8bac60d.png)

![image](https://user-images.githubusercontent.com/67769750/103207146-2dc3da00-4930-11eb-8df4-7bbeb8a1e201.png)

Is `margin-left` here the margin from the texts `Model` and `Load` to the left border of their containers? If that's right, please change the value to that used in the mockup.

Reply:
it's a distances from the left side of icon to text. But thanks i have updated the exactly px need to fix with it